.changes/unreleased/Breaking Changes-20230721-125806.yaml

```diff
@@ -0,0 +1,7 @@
+kind: Breaking Changes
+body: Update dbt-core to 1.6.0rc1
```

Comment:
Shouldn't you mention that people using `materialized_view` materialization should change that into `materializedview`?

Reply:
why even introduce such breaking change?

Reply:
This change is about [Relation type name](https://docs.getdbt.com/reference/dbt-classes#relation), not materialization name. [Materialization name](https://github.com/starburstdata/dbt-trino/blob/80e73a1f937e1bfe19c778b26aec0e7abcb9468e/dbt/include/trino/macros/materializations/materialized_view.sql#L1) is not changed - still `materialized_view`

This change is needed to be applied along with adjusting to 1.6, as this relation type name change was introduced in `dbt-core` [here](https://github.com/dbt-labs/dbt-core/commit/2d237828ae2a94f39e3ce8e4680a84ac22b6ead4#diff-22e2281122ce5401cfcd0746e181a653a06e56f1c47823333d1dc35304ce5495). Without our adjustment, we are receiving error:
```
E           dbt.exceptions.DbtRuntimeError: Runtime Error
E             'materializedview' is not a valid RelationType
```

So it could be breaking change if users wrote their own macros, in which they are checking if relation type equals to `materializedview`. Should I mention it? Where it should be mentioned, on slack while announcing new version?

Reply:
I think changelog is probably all right in this case as it seems rather an internal detail.

Reply:
I added new changie entry for that anyway, but right it is rather an internal detail